query-builder/src/test/java/com/datastax/oss/driver/api/querybuilder/update/UpdateFluentAssignmentTest.java

```diff
@@ -60,6 +60,16 @@ public void should_generate_map_value_assignment() {
         .hasCql("UPDATE foo SET features['color']=? WHERE k=?");
   }
 
+  @Test
+  public void should_generate_list_value_assignment() {
+    assertThat(
+            update("foo")
+                .setListValue("features", literal(1), bindMarker())
+                .whereColumn("k")
+                .isEqualTo(bindMarker()))
+        .hasCql("UPDATE foo SET features[1]=? WHERE k=?");
+  }
```

Comment:
missing test for map

Reply:
it's just above this test on line 64: `should_generate_map_value_assignment`.